kolena/dataset/evaluation.py

```diff
@@ -197,12 +196,33 @@ def upload_results(
                     validate_dataframe_ids(df_result, id_fields)
                     validate_dataframe_have_other_columns_besides_ids(df_result, id_fields)
                     id_column_validated = True
+                total_rows += df_result.shape[0]
                 df_results = _process_result(config, df_result, id_fields)
                 upload_data_frame(df=df_results, batch_size=BatchSize.UPLOAD_RECORDS.value, load_uuid=load_uuid)
 
-    response = _upload_results(model, load_uuid, existing_dataset.id)
+    response = _send_upload_results_request(model, load_uuid, existing_dataset.id)
     log.info(
         f"uploaded test results for model '{model}' on dataset '{dataset}': "
+        f"There are a total of {total_rows} rows in the results input"
```

Comment:
Prefer to keep log succinct, suggest `{} uploaded, {} inserted, {} updated`

Reply:
sounds good

Reply:
Do we also want to include result link to web UI?

Reply:
We do have a task of logging the result url, do you mean we should return the result url in the response?

Reply:
I meant only logging the result url, fine for the other ticket to cover it.

Reply:
I still think this is valuable to see `{} uploaded, {} inserted, {} updated`. I don't know a good way to get these info for a result file upload from sdk on the UI, plus this is one less click